include/klee/Expr.h

```diff
@@ -1080,7 +1080,8 @@ class ConstantExpr : public Expr {
   }
 
   static ref<ConstantExpr> create(uint64_t v, Width w) {
-    assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
+    if (w <= 64)
```

Comment:
With check you mean the `if` or the assertion without any `if`? In the latter case, we will still observe assertion failures.